LogicalTypes.md

```diff
@@ -256,6 +256,24 @@ The primitive type is a 2-byte `FIXED_LEN_BYTE_ARRAY`.
 
 The sort order for `FLOAT16` is signed (with special handling of NANs and signed zeros); it uses the same [logic](https://github.com/apache/parquet-format#sort-order) as `FLOAT` and `DOUBLE`.
 
+### FIXED_SIZE_LIST
+
+The `FIXED_SIZE_LIST` annotation represents a fixed-size list of elements
+of a primitive data type. It must annotate a `FIXED_LEN_BYTE_ARRAY` primitive type.
```

Comment:
As written, the elements can themselves be arrays. Is this intended? Or should it be "non-array primitive data type"?

Reply:
I didn't really consider the possibility of elements being arrays and I think non-array limitation makes sense. Changed to:
```markdown
The `FIXED_SIZE_LIST` annotation represents a fixed-size list of elements
of a non-array primitive data type. It must annotate a `FIXED_LEN_BYTE_ARRAY` primitive type.
```